pkgs/top-level/all-packages.nix

```diff
@@ -20428,8 +20429,12 @@ in
   djview = libsForQt5.callPackage ../applications/graphics/djview { };
   djview4 = pkgs.djview;
 
-  dmenu = callPackage ../applications/misc/dmenu { };
-  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix { };
+  dmenu = callPackage ../applications/misc/dmenu {
+    patches = config.dmenu.patches or [];
+  };
+  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix {
```

Comment:
The function returned from that file doesn't even have `patches` in it's argset, are you sure this works?

I haven't tested it though, as I'm doing the comments from a mobile.

Reply:
Just see it in the error of the GitHub action...

It does indeed not evaluate because of this:

https://gist.github.com/GrahamcOfBorg/0f56c0ebd42091febfe5410583975bf6#file-output-path-comparison-L9